src/QuadraticProgram/QuadraticProgram.jl

```diff
@@ -9,6 +9,8 @@ import LazyArrays
 import IterativeSolvers
 
 import DiffOpt
+import LinearSolve
+const LS = LinearSolve
```

Comment:
Let's not introduce `LS` if we use it only 3 times, it just decreases readability IMO